### DISCUSSION TITLE
they dont work

### DISCUSSION THREAD

**Discussion by asherbear1243** [TrickJoker312/Unblocked-Sites#40] [General]
they hardly work with gogardian

**Comment by J7DT2023** [comment]
> they hardly work with gogardian

same bro smh

**Comment by TrickJoker312** [comment]
Hey!

My school uses Securly, so i don't really know much about the type of links that WOULD work with goguardian. Can you please provide more insight about this topic? Thanks!

**Comment by J7DT2023** [comment]
just found out that my school uses both securly and goguardian so what ever passes by securly gets blocked by goguardian and vice versa  so fml

**Comment by asherbear1243** [comment]
is how my school is securly and gogardian

**Reply by J7DT2023** [reply]
I found a site that worked for me care to try it? to see if it works for u?

**Comment by asherbear1243** [comment]
sure i will try it

**Comment by J7DT2023** [comment]
sorry for taking so long but here's the link bro hope it gives you some sense of freedom if it works [[nightlight.zapto.org](https://nightlight.zapto.org/)
(url)

**Reply by crisjonks** [reply]
instead use [this link](gointerstellar.app)
